core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -4246,13 +4286,12 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 uint32 flags;
                 mem_offset_t offset, addr;
 
-                read_leb_uint32(frame_ip, frame_ip_end, flags);
+                read_leb_align(frame_ip, frame_ip_end, flags);
                 read_leb_mem_offset(frame_ip, frame_ip_end, offset);
                 addr = POP_MEM_OFFSET();
                 CHECK_MEMORY_OVERFLOW(4);
                 PUSH_I32(LOAD_I32(maddr));
                 CHECK_READ_WATCHPOINT(addr, offset);
```

Comment:
maybe you need to adapt watchpoint.
i suspect you might need to adapt lldb and its protocol as well.

in the mean time i guess you can make it perform watchpoint processing only when memidx == 0.

Reply:
Indeed there are more work needs to be done to support multi-memory interp debugging. So currently in the config_common.cmake I enforce to disable the interp debugging feature when enabling multi-memory.